terms/WasAttributedTo

```diff
@@ -0,0 +1,24 @@
+{
+    "@context": "../contexts/base",
+    "@graph": [
+        {
+            "@id": "nidm:subject_id",
+            "schema:domainIncludes": {
+                "@id": "reproschema:WasAttributedTo"
+            }
+        },
+        {
+            "@id": "reproschema:Response",
```

Comment:
this file should be `reproschema:Participant`

Reply:
this id should be to itself no? `reproschema:WasAttributedTo` ?